code/modules/admin/admin_verbs.dm

```diff
@@ -617,16 +619,16 @@ var/list/admin_verbs_mod = list(
 	//writepanic("[__FILE__].[__LINE__] ([src.type])([usr ? usr.ckey : ""])  \\/client/proc/drop_bomb() called tick#: [world.time]")
 
 	var/turf/epicenter = mob.loc
-	var/list/choices = list("Small Bomb", "Medium Bomb", "Big Bomb", "Custom Bomb")
+	var/list/choices = list("Small Bomb (1,2,3)", "Medium Bomb (2,3,4)", "Big Bomb (3,5,7)", "Custom Bomb")
 	var/choice = input("What size explosion would you like to produce?") in choices
 	switch(choice)
 		if(null)
 			return 0
-		if("Small Bomb")
+		if("Small Bomb (1,2,3)")

```

Comment:
I think it's used. And those explosions DO have a flash value. I only added the dev/heavy/light values in the button name because that's what you generally care about when you make an explosion